1.1/learn/how-to-document-ballerina-code.md

```diff
@@ -64,7 +64,7 @@ The supported structure of documentation syntax is as follows:
 #            <return_parameter_description_line_2>
 ```
 
->**Tip:** As a best practice, do not add a full stop at the end of the first sentence of the parameter and return type descriptions. Instead, add a fullstop only at the end of a function description. For Example,
+>**Tip:** Always, add a full stop at the end of a function description. However, as a best practice, omit the full stop if you have only one sentence in the parameter and return type descriptions. If there are multiple sentences for a parameter or return type description, add the full stop at the end of the first sentence and in the succeding ones. For example,
 
 ```
```

Comment:
Should we use ballerina syntax highlighting using ( ```ballerina) in these pages instead of general syntax highlighting for ballerina code snippets?

Reply:
Done.